Index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Portfolio</title>
+    <!-- Css fie  -->
+    <link rel="stylesheet" href="Css/master.css" />
+    <!-- Render all elements normally -->
+    <link rel="stylesheet" href="Css/normalize.css" />
+    <!-- Font awsome library -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- header -->
+    <header>
+      <p>
+        <a href="#" class="logo">My Logo</a>
+     </p>
+      <nav class="menu">
+        <img src="/picture/Union.png" alt="" />
+      </nav>
+  </header>
+      </nav>
+    </header>
```

Comment:
- [ ] Kindly make sure you write very well your tag element, as you may notice how they are not logic here.
```suggestion
      </nav>
  </header>
```